app/modules/listings/components/listing_element_actions.svelte

```diff
@@ -45,16 +52,28 @@
         elements = oldElements
       })
   }
+
+  function toggleEditMode () {
+    isEditMode = !isEditMode
+  }
 </script>
+{#if isEditMode}
```

Comment:
[non-blocking]
```suggestion
{#if showEditModal}
```

Reply:
done in https://github.com/inventaire/inventaire-client/commit/d3d025a7cce87e7747df00df53771d57ebf2367b